### PR TITLE
Fixes #4520, added gigapascal to pressure units to fix broken unit

### LINKED ISSUE #4520: Conversions: Should not trigger on "gpa calculator"
Produces conversion from an undefined unit, resulting in a blank dropdown   

https://duckduckgo.com/?q=gpa+calculator

------
IA Page:  http://duck.co/ia/view/conversions 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @pjhampton

### PR DESCRIPTION
**Pull request by urohit011**
Added 'Gigapascle' unit for pressure for Conversions in conversions.js. 
## Description of new Instant Answer, or changes

Fixes #4520, Updated Conversion goodie to return proper results for queries requesting conversion from 'Gigapascal' unit.
## People to notify

---

Instant Answer Page: https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
Left some comments in the Issue. This is the wrong approach.

**Comment by moollaza** [comment]
@urohit011 actually, we can still accept this so `1 gpa to megapascal` works. More importantly we need to ignore the query "gpa calculator" though.

**Comment by moollaza** [comment]
@pjhampton any concerns? This LGTM :+1:

File: t/Conversions.t [hunk 1]
```diff
@@ -536,6 +536,14 @@ ddg_goodie_test(
             physical_quantity => 'pressure'
         })
     ),
+    '1 gpa to megapascal' => test_zci(
+        '', structured_answer => make_answer({
```
Comment (pjhampton) [inline t/Conversions.t]: Can you please add a test for Gbar aswell please?

**Comment by pjhampton** [comment]
@moollaza seems to be, but please deploy to beta just to make super sure.

**Comment by urohit011** [comment]
Sure thing, I'll add test for Ghar as well.

File: t/Conversions.t (hunk 1 above)
Comment (urohit011) [reply t/Conversions.t]: Sure thing, I'll add test for Gbar as well.

**Comment by urohit011** [comment]
Added test for Gbar.

**Comment by moollaza** [comment]
Deploying to beta..

**Comment by moollaza** [comment]
@urohit011 Thanks! This LGTM :+1:

**Pull request merged**